episodes/introduction.md

```diff
@@ -14,31 +14,104 @@ OVERVIEW
 
 :::{image} ../media/episodes/introduction/hub-overview.png
 :width: 100%
-:alt: Diagram showing an overview of a 2i2c Hub, including authentication, online content, custom environments, the community hub, cloud infrastructure, support and services.
+:alt: Diagram showing an overview of a 2i2c hub, including authentication, online content, custom environments, the community hub, cloud infrastructure, support and services.
+
+:::
+
+### Community Hub
+
+A hub provides an access point to interactive computing in the cloud for a user community. Access to the hub is via a URL of the following form
+
+```shell
+<hub-name>.<community-name>.2i2c.cloud
+```
+
+and the landing page contains a *Log in to continue* button (see example screenshot)
+
+:::{image} ../media/episodes/introduction/community-hub.png
+:width: 100%
+:alt: Screenshot of a hub landing page with a 'Log in to continue' button.
+
+:::
+
+### Authentication
+
+Access to the hub is controlled by Hub Champions and granted by adding a user's GitHub account to a special GitHub Team associated with the hub. A permitted user enters their GitHub credentials to log into the hub (see example screenshot)
+
+:::{image} ../media/episodes/introduction/authentication.png
+:width: 100%
+:alt: Screenshot of a Server Options page. Available machine types include Shared Small: 1-4 CPU, 8-32 GB, 
+
+:::
+
+### Custom environments
+
+Users can choose from several machine types with varying numbers of CPU cores and RAM available, and select their desired software environment using images.
+
+:::{image} ../media/episodes/introduction/custom-env-1.png
+:width: 100%
+:alt: Screenshot of a hub login page using GitHub for authentication. Server options available include Shared Small: 1-4 CPU, 8-32 GB RAM, Small: 4 CPU, 32 GB RAM, Medium: 16 CPU, 128 GB RAM and Large: 64 CPU, 512 GB RAM.
 
 :::
 
-## Exercises
+Images available by default include
+
+- Handbook Authoring (JupyterLab user interface) – installed Python packages include `jupyter-book, jupyterlab_myst, ghp-import, numpy, matplotlib`
+- [Jupyter DataScience](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-datascience-notebook) (JupyterLab user interface) – installed Python packages include `dask, h5py, pandas, scikit-learn, scipy, sympy`
+- [Rocker Geospatial](https://rocker-project.org/images/versioned/rstudio.html) (RStudio user interface) – installed R packages include `ncdf4, proj4, raster, rgdal, rgeos, sf, sp`
+- Other... – specify a custom JupyterLab/RStudio image
+
+### Online Content
+
+There are many ways to manage online content on the hub. For example 
+
+- users have access to their own filesystem and a home directory of up to 20 GB
+- Hub Champions can distribute data to all hub users in a shared directory
+- users can securely pull and push code to and from the hub using `gh-scoped-creds`.
```

Comment:
Can you add a little more explanation about what `gh-scoped-creds` is here? Or, in the other direction, trim this down to, simply

> users can securely pull and push code to and from the hub

Reply:
Replaced with the word 'GitHub' in 6ea1a55228360b622e4b576687304291102c2763